### PR TITLE
react-file-uploads

### PR DESCRIPTION
**Pull request by nhuphham**
Screen Shot 2022-09-21 at 10 02 44 PM

### CODE REVIEW

File: react-file-uploads/server/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const { caption } = req.body;
+  if (!caption) {
+    throw new ClientError(400, 'caption is a required field');
+  }
+  /**
+   * - create a url for the image by combining '/images' with req.file.filename
+   * - insert the "caption" and "url" into the "images" table
+   * - respond with the inserted row data
+   * - catch any errors
+   */
```
Comment (scbowler) [inline react-file-uploads/server/index.js:36]: This part needs to be completed as well.

**Review by scbowler — CHANGES_REQUESTED** [review]
Client side looks good. The server side code still needs to be completed. Let me know if you have any questions.

File: react-file-uploads/server/index.js (hunk 1 above)
Comment (nhuphham) [reply react-file-uploads/server/index.js:36]: Screen Shot 2022-09-29 at 5 41 52 PM

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**